src/Apps/Home/__tests__/HomeNewWorksFromGalleriesYouFollowRail.jest.tsx

```diff
@@ -0,0 +1,153 @@
+import { graphql } from "react-relay"
+import { setupTestWrapper } from "DevTools/setupTestWrapper"
+import { HomeNewWorksFromGalleriesYouFollowRailFragmentContainer } from "Apps/Home/Components/HomeNewWorksFromGalleriesYouFollowRail"
+import { HomeNewWorksFromGalleriesYouFollowRail_Test_Query } from "__generated__/HomeNewWorksFromGalleriesYouFollowRail_Test_Query.graphql"
+import { useTracking } from "react-tracking"
+
+jest.unmock("react-relay")
+jest.mock("react-tracking")
+
+const { getWrapper } = setupTestWrapper<
+  HomeNewWorksFromGalleriesYouFollowRail_Test_Query
+>({
+  Component: props => {
+    return (
+      <HomeNewWorksFromGalleriesYouFollowRailFragmentContainer
+        newWorksFromGalleriesYouFollowConnection={
+          props.me?.newWorksFromGalleriesYouFollowConnection!
+        }
+      />
+    )
+  },
+  query: graphql`
+    query HomeNewWorksFromGalleriesYouFollowRail_Test_Query
+      @relay_test_operation {
+      me {
+        newWorksFromGalleriesYouFollowConnection(first: 20) {
+          ...HomeNewWorksFromGalleriesYouFollowRail_newWorksFromGalleriesYouFollowConnection
+        }
+      }
+    }
+  `,
+})
+
+const trackEvent = jest.fn()
+
+beforeAll(() => {
+  ;(useTracking as jest.Mock).mockImplementation(() => ({ trackEvent }))
+})
+
+afterEach(() => {
+  trackEvent.mockClear()
+})
+
+describe("HomeNewWorksFromGalleriesYouFollowRail", () => {
+  it("renders correctly", () => {
+    const wrapper = getWrapper({
+      Me: () => meResponse,
+    })
+
+    expect(wrapper.text()).toContain("The World As I Found It")
+    expect(wrapper.html()).toContain(
+      "/artwork/francine-tint-the-world-as-i-found-it"
+    )
+  })
+
+  describe("tracking", () => {
+    it("tracks view all clicks", () => {
+      const wrapper = getWrapper()
+      wrapper.find("RouterLink").at(1).simulate("click")
+
+      expect(trackEvent).toBeCalledWith({
+        action: "clickedArtworkGroup",
+        context_module: "troveArtworksRail",
+        destination_page_owner_id: "932d0b13-3cf1-46d1-8e49-18b186230347",
+        destination_page_owner_slug: "curators-picks-emerging",
+        destination_page_owner_type: "collection",
+        type: "viewAll",
+        context_page_owner_type: "home",
+      })
+    })
+
+    it("tracks item clicks", () => {
+      const wrapper = getWrapper()
+      wrapper.find("RouterLink").at(2).simulate("click")
+
+      expect(trackEvent).toBeCalledWith({
+        action: "clickedArtworkGroup",
+        context_module: "newWorksByGalleriesYouFollowRail",
+        context_page_owner_type: "home",
+        destination_page_owner_id: "<Artwork-mock-id-1>",
+        destination_page_owner_slug: "<Artwork-mock-id-2>",
+        destination_page_owner_type: "artwork",
+        type: "thumbnail",
+      })
+    })
+  })
+})
+
+const meResponse = {
```

Comment:
Ditto here, re generating responses. Why create fixtures when they can be autogenerated by relay's testing tools? We should only create fixtures out of the fields that are needed directly within the test, or for things that help the component render in certain ways.

Reply:
Removed the fixture 👍